doc/v2/howto/capi/index_en.rst

```diff
@@ -1,6 +1,23 @@
 C-API Prediction Library
 ========================
 
+After we trained a neural network model, then we use it to make predictions. Prediction is a process of preparing input data and processing the model to get the result.
```

Comment:
trained -> **train**

Reply:
I think we should replace the word prediction with Inference. The title can also read C-API Inference Library.

Reply:
The sentence should be as follows:

After we train a neural network, we use it to do inference. Inference is the process of preparing input data and propagating it through the model to produce the result.